repository-ydb-common/src/main/java/tech/ydb/yoj/repository/ydb/exception/YdbClientInternalException.java

```diff
@@ -1,23 +0,0 @@
-package tech.ydb.yoj.repository.ydb.exception;
-
-import tech.ydb.yoj.repository.db.exception.RepositoryException;
-import tech.ydb.yoj.repository.db.exception.RetryableException;
-import tech.ydb.yoj.repository.db.exception.UnavailableException;
-import tech.ydb.yoj.util.lang.Strings;
-import tech.ydb.yoj.util.retry.RetryPolicy;
-
-/**
- * Internal YDB SDK exception, caused by a transport failure, internal authorization/authentication error etc.
- */
-public final class YdbClientInternalException extends RetryableException {
```

Comment:
??? bring it back maybe, as a subclass of YdbConditionallyRetryableException ???